src/helpers/parseJwt.js

```diff
@@ -0,0 +1,6 @@
+export const parseJwt = token => {
+    const base64Url = token.split('.')[1];
```

Comment:
Please use a library to parse JWT. Maybe this one https://github.com/auth0/node-jsonwebtoken

Reply:
Done, @italolelis